satzung.md

```diff
@@ -20,15 +20,15 @@ Die Organe der Fachschaft sind:
 
 ## § 3 Vollversammlung
 
-(1) Die Vollversammlung der Fachschaft ist das oberste beschlussfassende Organ der Fachschaft. In der Vollversammlung haben alle Mitglieder der Fachschaft Sitz und Stimme.
+(1) Die Vollversammlung der Fachschaft ist das oberste beschlussfassende Organ der Fachschaft. In der Vollversammlung haben alle Mitglieder der Fachschaft Sitz und Stimme sowie ein Rede- und Antragsrecht.
 
-(2) In jedem Studienjahr soll eine ordentliche Vollversammlung stattfinden. Für die Durchführung ist der Fachschaftsrat verantwortlich.
+(2) In jedem Studienjahr soll eine ordentliche Vollversammlung stattfinden. Ordentliche Vollversammlungen werden durch den Fachschaftsrat einberufen.
```

Comment:
Will man hier festlegen, dass eine ordentliche Vollversammlung nur die einmal pro Jahr stattfindende sein kann? Das hat direkte Auswirkungen auf die Diskussion zur Satzungsänderung unten (im Bezug darauf ob man unter dem Jahr eine weitere ordentliche einberufen kann)
@felixrindt pls do discussion at next Sitzung

Reply:
Da sind wir wieder beim Thema "Soll-Bestimmungen". Das schließt nicht aus, dass mehrere ordentliche Vollversammlungen pro Jahr stattfinden. Man könnte natürlich konkreter schreiben, dass "mindestens eine ordentliche VV" stattfinden soll.

Reply:
der Punkt von "ordentlich" ist ja dass sie die eine normale VV ist die man halt hat. Deswegen will ich ja unten auch das "ordentlich" nicht.

Reply:
@hendraet Genau das würde ich aber ausschließen, sonst ist halt der Begriff von ordentlich ad absurdum geführt und wir können uns die Unterscheidung auch sparen. 
Dafür muss man dann natürlich die Satzung auf einer außerordentlichen ändern können.

Reply:
Hab die Diskussion unten nicht wirklich mitbekommen. Jetzt seh ich dein Problem. 
Für Satzungsänderungen halte ich die längere Ankündigungszeit wichtig, weshalb ich so etwas nur auf ordentlichen VVs machen würde. Deswegen würde ich es nicht auf eine ordentlich VV beschränken. Ich glaube nicht, dass es wirklich mal dazu kommt, dass sich ein FSR aus unlogischen Gründen weigert und wenn doch kann man den ja auf ner AVV abwählen.

Reply:
@hendraet you got a "nicht" zuviel?

Reply:
Ne, das stimmt schon. Ich wollte sagen: "Deswegen würde ich es nicht auf **eine** ordentlich VV beschränken"

Reply:
ah. Ja dann ändert man die Bedeutung von "ordentlich". Vorher meinte "ordentlich" ja sowas wie "turnusgemäß"...